src/python/generate_singa_wrapper.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+!/usr/bin/env bash
```

Comment:
bash is not available on Windows, need to move these commands to cmake in the future.